src/components/Popover/types.ts

```diff
@@ -38,8 +38,8 @@ type PopoverProps = BaseModalProps &
         /** Whether we want to show the popover on the right side of the screen */
         fromSidebarMediumScreen?: boolean;
 
-        /** Whether handle navigation back when modal show. */
-        shouldHandleNavigationBack?: boolean;
```

Comment:
Why did you remove this props? We're stilling using it in our Popover base component right?